validators/payable_transactions.go

```diff
@@ -62,31 +62,9 @@ func TransactionsArePayable(companyNumber string, txs []models.TransactionItem)
 			return nil, ErrTransactionDoesNotExist
 		}
 
-		if val.IsPartPaid() {
```

Comment:
Attempt to reduce code complexity by grouping similar logic checks into one Go function